### DISCUSSION TITLE
Osu!Lazer: Changed folders and all my beatmaps are gone. (Solved)

### DISCUSSION THREAD

**Discussion by AxelGIThu** [ppy/osu#31365] [Q&A] [answered]
A bit of context, I changed to a new pc and imported all the beatmaps using BeatmapExporter and a pendrive that I had (3300+ about 32gb) and it took a long while (4hs+) because my other pc was slow.

All worked fine, played a few songs and closed the game, I noticed that the Osu! folder was on my OS partition so I used the "change folder location..." option and selected my other partition, then since it was late I checked that my beatmaps where still there (they were) and went to sleep.
Next morning, boot it up to play some more and all my songs were gone.
help 😢

**Comment by peppy** [comment, marked as answer]
- Locate which folder on your PC is large
- Use the "change folder location" button to select that folder.

That said, I have no idea what `BeatmapExporter` is, so you may have broken something using external tooling, not sure.

**Reply by AxelGIThu** [reply]
Thanks for the fast response!

After checking what you said, I saw an "osu!" folder with 30gb on my recycle bin, I restaured it and then changed the folder with the in-game option as you said, all fixed now.

This was most probably a me error, so im sorry for bothering, have a nice day and happy new year!
